js/_website/src/lib/components/DemosLite.svelte

```diff
@@ -52,22 +63,13 @@
 			controlPageTitle: false,
 			appMode: true
 		});
+		const debounce_timeout = 1000;
+		debounced_run_code = debounce(controller.run_code, debounce_timeout);
+		debounced_install = debounce(controller.install, debounce_timeout);
+
 		mounted = true;
 	});
 
-	function update(code: string, requirements: string[]) {
```

Comment:
tbh, I couldn't get why this double function calling could suppress errors, but the problem was fixed as a result of the refactoring.